xtraplatform-string-templates/src/test/groovy/de/ii/xtraplatform/stringtemplates/domain/StringTemplateFiltersSpec.groovy

```diff
@@ -0,0 +1,170 @@
+package de.ii.xtraplatform.stringtemplates.domain
+
+import spock.lang.Ignore
+import spock.lang.Specification
+
+class StringTemplateFiltersSpec extends Specification {
+
+    @Ignore
+    def 'All arguments are null'() {
+        when:
+            String result = StringTemplateFilters.applyTemplate(null, null, null, null)
+        then:
+            result.isEmpty()
+    }
+
+    @Ignore
+    def 'Template is null, value is present'() {
+        given:
+            String template = null
+            String value = "foobar"
+        when:
+            String result = StringTemplateFilters.applyTemplate(template, value)
+        then:
+            result.isEmpty()
+    }
+
+    @Ignore
+    def 'Value is null, template is present'() {
+        given:
+            String template = "{{value | replace:'ABC':' '}}"
+            String value = null
+        when:
+            String result = StringTemplateFilters.applyTemplate(template, value)
+        then:
+            result.isEmpty()
+    }
+
+    @Ignore
+    def 'Template is empty, value is present'() {
```

Comment:
This currently fails, so I added `@Ignore`. But I agree with you that this test should succeed. So can you adjust `applyTemplate` in `StringTemplateFilters`, so that it first checks for an empty template and returns the value in this case?